Source/ALS/Private/AlsAnimationInstance.cpp

```diff
@@ -384,7 +384,7 @@ void UAlsAnimationInstance::RefreshLook()
 	{
 		// Offset the angle to keep it relative to the movement base.
 
-		Look.WorldYawAngle = FRotator3f::NormalizeAxis(Look.WorldYawAngle + MovementBase.DeltaRotation.Yaw);
+		Look.WorldYawAngle = FRotator3f::NormalizeAxis(Look.WorldYawAngle + static_cast<float>(MovementBase.DeltaRotation.Yaw));
```

Comment:
Replace all `static_cast<float>()` with `UE_REAL_TO_FLOAT`.

Reply:
Ok, but the only benefit I see is `UE_REAL_TO_FLOAT` enable us to choose whether to use `static_cast`, in what case shouldn't we use it?

Reply:
From the comment to this macro:

> Use to make any narrowing casts searchable in code when it is updated to work with a 64 bit count/range

Reply:
Isn't `static_cast<float>` a narrowing cast and could be searched?